src/IssueViz/TableControls/VSTableEventProcessorProvider.cs

```diff
@@ -39,12 +40,12 @@ namespace SonarLint.VisualStudio.IssueVisualization.TableControls
     [DataSource(SonarLintTableControlConstants.ErrorListDataSourceIdentifier)]
     internal class VSTableEventProcessorProvider : ITableControlEventProcessorProvider
     {
-        private readonly IIssueTablesSelectionMonitor issueTableSelectionMonitor;
+        private readonly IIssueSelectionService selectionService;
```

Comment:
The old interface had the exact "Select" functionality and it was also a general purpose one, so I just replaced it with the new one.